assets/src/components/Lightbox/Lightbox.js

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import PhotoSwipe from '../../../../node_modules/photoswipe/dist/photoswipe.js';
+import PhotoSwipeUI_Default from '../../../../node_modules/photoswipe/dist/photoswipe-ui-default.js';
+
+export const Lightbox = props => {
+  let photoSwipeElement = useRef(null);
+
+  const options = {
+    index: props.index || 0,
+    closeOnScroll: false,
+    history: false
+  };
+
+  const [currentIndex, setCurrentIndex] = useState(options.index);
+
+  useEffect(() => {
+    const photoSwipe = new PhotoSwipe(photoSwipeElement, PhotoSwipeUI_Default, props.items, options);
+
+    if (!photoSwipe) {
+      return;
+    }
+
+    if (!props.isOpen) {
```

Comment:
This currently won't be run if  `props.isOpen` changes, it's not in the deps. But not suggesting to add it to the same effect, it should probably be a separate one, if we still need to use an effect for that.

Reply:
Moved it outside of the effect.